lib/scss/inc/_variables.scss

```diff
@@ -30,21 +29,29 @@ $link-color: #2d91e6;
 
 // Font
 $font-size-base: 18px;
-$font-family: 'Source Sans Pro', sans-serif;
-$font-weight-bold: 600;
+$font-family: 'Roboto', sans-serif;
```

Comment:
I thought we decided on using Source Sans Pro as per Jayshree's feedback and we made the changes on #7 

Also due to this change, as we don't have the `Roboto` font loaded on `index.erb`, we just see the `sans-serif` font on the browser

Reply:
Yep i think we did Source sans pro for the miniohq/gluegun